src/ImageSharp.Web/Processors/ResizeWebProcessor.cs

```diff
@@ -182,10 +213,98 @@ or ExifOrientationMode.RightBottom
             => parser.ParseValue<ResizeMode>(commands.GetValueOrDefault(Mode), culture);
 
         private static AnchorPositionMode GetAnchor(
+            ushort orientation,
             CommandCollection commands,
             CommandParser parser,
             CultureInfo culture)
-            => parser.ParseValue<AnchorPositionMode>(commands.GetValueOrDefault(Anchor), culture);
+        {
+            AnchorPositionMode anchor = parser.ParseValue<AnchorPositionMode>(commands.GetValueOrDefault(Anchor), culture);
+            if (orientation is >= ExifOrientationMode.Unknown and <= ExifOrientationMode.TopLeft)
+            {
+                return anchor;
+            }
+
+            /*
```

Comment:
I had to use a notepad for this bit. I think my logic is sound but could do with a second opinion.